src/test_opencost_parquet_exporter.py

```diff
@@ -90,6 +90,27 @@ def test_get_config_no_window_end(self):
             self.assertNotIn('s3_bucket', config)
             self.assertEqual(config['params'][0][1], window)
 
+    def test_get_config_with_parquet_prefix_env_var(self):
```

Comment:
can you add a case to verify that by default prefix is generated using the current  year/month/day format ?

Reply:
Done ✅